src/server/pfs/fuse/filesystem.go

```diff
@@ -155,6 +155,10 @@ func (d *directory) Create(ctx context.Context, request *fuse.CreateRequest, res
 		size:      0,
 	}
 	if err := localResult.touch(); err != nil {
+		// Check if its a write on a finished commit:
+		if readOnly := strings.Contains(err.Error(), "has already been finished"); readOnly {
```

Comment:
I generally like the standard of defining predicates in the same file as the errors themselves. This one could maybe be called `IsPermissionError` or something.